moped-editor/src/views/projects/projectView/ProjectComments.js

```diff
@@ -271,8 +280,9 @@ const ProjectComments = () => {
                 <List className={classes.root}>
                   {data.moped_proj_notes.map((item, i) => {
                     let isNotLastItem = i < data.moped_proj_notes.length - 1;
-                    let editableComment = (userSessionData.user_id === item.added_by_user_id ||
-                                           userHighestRole === "moped-admin");
+                    let editableComment =
```

Comment:
from here down is just prettier

Reply:
any chance we could have those two variables made const?

Reply:
sure, the only change was me running prettier on the file, ill change them to const.